programs/utilities/sscoreboard.sc

```diff
@@ -0,0 +1,86 @@
+//statistic scoreboard interface by Lokdora
+//dynamically create scoreboard based on statistics and assigned it with correct value (when player is online)
+//options to distinguish bots from other players
+//due to the scoreboard value cannot be assigned when player is offline, it is better to use this script to create scoreboard objectives for mid/long-term use (or singleplayer world).
+
+
+__config() -> {
+    'stay_loaded' -> 'true',
+	'scope' -> 'global',
+	'commands' -> {
+		'' -> 'help',
+		'clear' -> 'clear',
+		'<stat> <displayName>' -> ['create','sidebar',false],
+		'<stat> <displayName> <displaySlot>' -> ['create',false],
+		'<stat> <displayName> <displaySlot> <includeBots>' -> 'create'
+	},
+	'arguments' -> {
+		'stat' -> {'type' -> 'term'},
+		'displayName' ->{'type' -> 'string'},
+		'displaySlot' ->{'type' -> 'scoreboardslot'},
+		'includeBots' -> {'type' -> 'bool'}
+	}
+};
+
+entity_load_handler('player',_assign(p,new) -> (

```

Comment:
Does that even work? Also use `__on_player_connects` event, which is more appropriate here.

Reply:
The event handler works well, I have tested it for a lot of times. And I have tried __on_player_connects at the beginning, but it is not triggered when bot is summoned. I don't know whether it's a bug or fake players don't actually connect to the server.

Reply:
Oh I ran more tests and found the __on_player_connects actually works, it's just the output of print function isn't showed to me. I have switched to this function instead.